deny.toml

```diff
@@ -25,6 +25,7 @@ allow = [
   "CC0-1.0",
   "Unicode-3.0",
   "Zlib",
+  "CDLA-Permissive-2.0",
```

Comment:
required for https://github.com/rustls/webpki-roots#license which we pick up for getting the certs for rustls-tls